src/controller/python/chip/ChipStack.py

```diff
@@ -212,7 +212,7 @@ def logFunct(timestamp, timestampUSec, moduleName, logCat, message):
             moduleName = ChipUtility.CStringToString(moduleName)
             message = ChipUtility.CStringToString(message)
             if self.addModulePrefixToLogMessage:
-                message = "WEAVE:%s: %s" % (moduleName, message)
+                message = "CP:%s: %s" % (moduleName, message)
```

Comment:
Why not CHIP: ?

Reply:
some sort of wrong pressed keys in vim I guess. it should have been chip. Done.